src/dynamic-page.scss

```diff
@@ -0,0 +1,346 @@
+    }
+
+    &--collapsed {
+      padding-bottom: 1rem;
+    }
+
+    &[aria-hidden="true"] {
+      display: none;
+    }
+  }
+
+  &__title-container {
+    @include fd-reset();
+
+    @include fd-flex-vertical-center() {
+      justify-content: flex-start;
+    }
+  }
+
+  .#{$fd-namespace}-breadcrumb.#{$block}__breadcrumb {
+    padding: 0.5rem 0 0.25rem 0;
+
+    &--s {
+      padding: 0;
+    }
+  }
+
+  &__breadcrumb-container {
+    @include fd-reset();
+
+    @include fd-flex-vertical-center() {
+      justify-content: space-between;
+    }
+  }
+
+  &__title, .#{$fd-namespace}-link.#{$block}__title {
+    color: var(--sapGroup_TitleTextColor);
+    font-size: var(--sapFontHeader3Size);
+    padding: 0.3125rem 0 0 0;
+
+    &--collapsed {
+      @include fd-ellipsis();
+    }
+
+    // tablet spec
+    &--m {
+      padding: 0.6875rem 0 0 0;
+    }
+  }
+
+  &__subtitle {
+    @include fd-reset();
+
+    color: var(--sapContent_LabelColor);
+    font-size: var(--sapFontSize);
+    margin-top: 0.25rem;
+  }
+
+  &__title-content {
+    @include fd-reset();
+
+    padding: 0 0 0 1rem;
+
+    @include fd-rtl() {
+      padding: 0 1rem 0 0;
+    }
+  }
+
+  &__toolbar {
+    padding: 0 0 0 1rem;
+    margin-left: auto;  // align toolbar to right in a flex layout
+
+    @include fd-rtl() {
+      margin-left: 0;
+      margin-right: auto;
+    }
+
+    &--actions {
+      padding: 0;
+    }
+  }
+
+  &__summarized-title-area {
+    @include fd-reset();
+
+    @include fd-flex-vertical-center() {
+      justify-content: space-between;
+    }
+
+    @include fd-fiori-focus(0.0625rem);
+
+    word-wrap: break-word;
+    background: var(--sapObjectHeader_Background);
+    box-shadow: $fd-dynamic-page-box-shadow;
+    height: 2rem;
+
+    &--s {
+      @include responsive-page(0.5rem);
+    }
+
+    &--m,
+    &--l {
+      @include responsive-page(2rem);
+    }
+
+    &--xl {
+      @include responsive-page(3rem);
+    }
+
+    &--no-shadow {
+      @include no-shadow();
+    }
+
+    &[aria-hidden="true"] {
+      display: none;
+    }
+  }
+
+  &__summarized-title {
+    color: var(--sapGroup_TitleTextColor);
+    font-size: var(--sapFontHeader5Size);
+    padding: 0;
+    line-height: 2rem;
+
+    &--collapsed {
+      @include fd-ellipsis();
+    }
+  }
+
+  &__header {
+    // contains B and B1
+    @include fd-reset();
+
+    background-color: $fd-dynamic-page-background-color;
+    padding: $fd-dynamic-page-header-padding-top-bottom 0;
+
+    &--transparent-bg {
+      background-color: var(--sapDynamicPage_Title_Header_Transparent_Background);
+    }
+
+    &--s {
+      @include responsive-page($fd-dynamic-page-header-padding-top-bottom, 0.5rem);
+    }
+
+    &--m,
+    &--l {
+      @include responsive-page($fd-dynamic-page-header-padding-top-bottom, 2rem);
+    }
+
+    &--xl {
+      @include responsive-page($fd-dynamic-page-header-padding-top-bottom, 3rem);
+    }
```

Comment:
I suggest to keep `--s/m/l/xl` in `fd-dynamic-page` block. Just to keep same spacing for every element inside. Also it will reduce effort that developer has to put to create desired look

Reply:
Hi @JKMarkowski I did try to have that initially. But there are separate top/bottom paddings specified for title-area/content/header, so even if I did have left-right spacing handled at page level, the top-bottom ones wouldn't be handled alongwith it and would have to be written separately. Even the bottom padding changes for title area depending on whether it is collapsed or not.
Since the spec also views the spacings as separate(they even mention the left/right spacing separately rather than on the page level), I thought it better to align to that.
![image](https://user-images.githubusercontent.com/53509521/93199732-5bfc2000-f76c-11ea-9e71-813d882348f3.png)

Reply:
i think we should align the naming for diff. inflections like in https://github.com/SAP/fundamental-styles/blob/0cdb89a25689387494d53cae0a3bf29941596f00/src/form-layout-grid.scss#L148
`lg`,...

Reply:
I guess that was a general comment requiring more discussion? I see quite a few components like action bar, dialog, tabs, page, shellbar use the --s, --m, --l approach.